databaseAnnotator/src/main/resources/application.properties.EXAMPLE

```diff
@@ -5,6 +5,7 @@ databaseannotator.pass=
 databaseannotator.driver=com.mysql.jdbc.Driver
 databaseannotator.connection_string=jdbc:mysql://<HOST>/<DATABASE>
 
-genomenexus.hgvs=http://<HOST>:<PORT>/variant_annotation/hgvs/
+genomenexus.hgvs=<HOST>/variant_annotation/hgvs/
+genomenexus.xrefs=<HOST>/xrefs/
 genomenexus.isoform_query_parameter=isoformOverrideSource
-genomenexus.hotspot_parameter=cancerHotspots
+genomenexus.enrichment_fields=hotspots,mutation_assessor
```

Comment:
Never mind .. I see that there are 2 properties files involved #42 changed annotationPipeline/src/main/resources/application.properties.EXAMPLE
this changes
databaseAnnotator/src/main/resources/application.properties.EXAMPLE

There are also 2 application-test.properties.EXAMPLE files which  might need to be updated.